src/popup/components/Modals/PasswordLogin.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <Modal
+    class="password-login"
+    from-bottom
+    centered
+  >
+    <div class="content-wrapper">
+      <div class="icon-wrapper">
+        <IconBoxed
+          :icon="LockIcon"
+          bg-colored
+          bg-more-transparent
+          icon-padded
+        />
+      </div>
+      <div class="info">
+        <h3 class="text-heading-4 heading">
+          {{ $t('pages.secureLogin.enterPassword') }}
+        </h3>
+
+        <div class="text-description">
+          {{ $t('pages.secureLogin.toUnlock') }}
+        </div>
+      </div>
+      <InputPassword
+        v-model="password"
+        data-cy="password"
+        autofocus
+        class="password-input"
+        :placeholder="$t('pages.secureLogin.login.placeholder')"
+        :label="$t('pages.secureLogin.login.label')"
+        :message="isAuthFailed ? $t('pages.secureLogin.login.error') : null"
+        @keydown.enter="login"
```

Comment:
Instead I suggest to wrap the input and the button with `<form>` element and replace the `@keydown` and `@click` with `@submit` event on the form.

Reply:
Here's Chrome complaining about this:
![image](https://github.com/user-attachments/assets/6c92abad-e02e-49f8-b7db-682bb5eac3bd)